libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tab.component.ts

```diff
@@ -38,7 +38,7 @@ export class SkyVerticalTabComponent implements OnInit, OnDestroy {
    * Indicates whether the tab is active when the tabset loads.
    */
   @Input()
```

Comment:
A note: there are a couple of changes in two vertical tab files in order to allow for the sectioned form work. The rest of the vertical tab strict mode work will be done in a follow up PR